tests/roles/stop_remaining_services/tasks/main.yaml

```diff
@@ -0,0 +1,69 @@
+- name: set shell vars for stopping other services
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.set_fact:
+    stop_other_services_shell_vars: |
+      CONTROLLER1_SSH="{{ controller1_ssh }}"
+      CONTROLLER2_SSH="{{ controller2_ssh }}"
+      CONTROLLER3_SSH="{{ controller3_ssh }}"
+      declare -A computes
+      computes=(
+      ["standalone.localdomain"]="{{ edpm_node_ip }}"
+      )
+
+- name: stop compute services
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ stop_other_services_shell_vars }}
+
+    ComputeServicesToStop=(
+                    "tripleo_nova_compute.service"
+                    "tripleo_nova_libvirt.target"
+                    "tripleo_nova_migration_target.service"
+                    "tripleo_nova_virtlogd_wrapper.service"
+                    "tripleo_nova_virtnodedevd.service"
+                    "tripleo_nova_virtproxyd.service"
+                    "tripleo_nova_virtqemud.service"
+                    "tripleo_nova_virtsecretd.service"
+                    "tripleo_nova_virtstoraged.service")
+
+    PacemakerResourcesToStop=(
+                    "galera-bundle"
+                    "haproxy-bundle"
+                    "rabbitmq-bundle")
+
+    echo "Disabling systemd units and cleaning up for compute services"
+    for i in "${!computes[@]}"; do
+        SSH_CMD="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@${computes[$i]}"
```

Comment:
CI is failing because the ssh key is not in that path:
```
SH_CMD='ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100'\n+ for service in ${ComputeServicesToStop[*]}\n+ echo 'Stopping the tripleo_nova_compute.service in compute standalone.localdomain'\n+ ssh -i '~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa' root@192.168.122.100 sudo systemctl is-active tripleo_nova_compute.service\nWarning: Identity file /home/zuul/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa not accessible: No such file or directory.\n+ ssh -i '~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa' root@192.168.122.100 sudo systemctl disable --now tripleo_nova_compute.service\nWarning: Identity file /home/zuul/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa not accessible: No such file or directory.
```
from https://logserver.rdoproject.org/25/225/26cfb47fdfd363007d741f0310b2768f2782c5f0/github-check/data-plane-adoption-OSP-17-to-extracted-crc/61ac0d8/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_with_ceph_out_2023-12-20T11:48:25EST.log
the right path in CI would be `~/src/github.com/openstack-k8s-operators/install_yamls/out/edpm/ansibleee-ssh-key-id-rsa`, but we expose a variable `install_yamls_path` that could be used here, see https://logserver.rdoproject.org/25/225/26cfb47fdfd363007d741f0310b2768f2782c5f0/github-check/data-plane-adoption-OSP-17-to-extracted-crc/61ac0d8/controller/data-plane-adoption-tests-repo/data_plane_adoption/vars.yaml

Reply:
thank you for looking into this. I've yet finished testing this locally, the reason it fails is

Not Found for url: https://trunk.rdoproject.org/rhel9-antelope/current-podified/delorean.repo"

I need to trick the repo-setup with adding an os type centos9 parameter, as it fails to lookup RDO for rhel

I'll update the ssh keys paths as well, just hasn't to look into the "journey" of that override to implement it :)
Anyhow, ssh commands seems working anyway

Reply:
Done. Let's see if it works. On my dev setup, at some point during the erasure of python3-tripleoclient dependencies, the node loses SSH access, until rebooted

Reply:
hm, it gives:
```
Warning: Identity file /home/zuul/src/github.com/openstack-k8s-operators/install_yamls//out/edpm/ansibleee-ssh-key-id_rsa not accessible: No such file or directory.
```

Reply:
this missinig ssh key file looks benign though, and things do work in CI. We can fix this later.